examples/one_degree_simulation.jl

```diff
@@ -44,8 +44,8 @@ grid = ImmersedBoundaryGrid(underlying_grid, GridFittedBottom(bottom_height);
 #
 # We include temperature and salinity surface restoring to ECCO data from 1993 thoughout the water column.
 
-start_date = DateTime(1993, 1, 1)
-stop_date = DateTime(1993, 12, 1)
+start_date = DateTime(1993, 6, 1)
+stop_date = DateTime(1994, 5, 1)
```

Comment:
why?

Reply:
because then the solution won't be biased to heat up as much at the beginning (eg starting during northern hemisphere winter)